tests/test_helpers.h

```diff
@@ -21,3 +21,34 @@ static inline bool compare_and_print(size_t len, uint8_t* comp, uint8_t* exp) {
     printf("**FAILED**\n");
   return ok;
 }
+
+
+#if defined(__x86_64__) || defined(_M_X64)
+typedef uint64_t cycles;
+
+static __inline__ cycles cpucycles_begin(void)
+{
+  uint64_t rax,rdx,aux;
+  asm volatile ( "rdtscp\n" : "=a" (rax), "=d" (rdx), "=c" (aux) : : );
+  return (rdx << 32) + rax;
+}
+
+static __inline__ cycles cpucycles_end(void)
+{
+  uint64_t rax,rdx,aux;
+  asm volatile ( "rdtscp\n" : "=a" (rax), "=d" (rdx), "=c" (aux) : : );
+  return (rdx << 32) + rax;
+}
+
+static inline void print_time(uint64_t count, clock_t tdiff, cycles cdiff){
```

Comment:
why the copy-paste of a definition here? sounds like print_time doesn't need to be behind an ifdef